<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DealyedSimulationDialog</class>
 <widget class="QDialog" name="DealyedSimulationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delayed Simluation Speed</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="lblSpeed">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="speedSlider">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lblInstrPer">
         <property name="text">
          <string>Instruction per</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSpeedVal">
         <property name="text">
          <string>0 milliseconds    </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DealyedSimulationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DealyedSimulationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
